pallets/time-release/src/lib.rs

```diff
@@ -171,20 +169,22 @@ pub mod module {
 	>;
 
 	#[pallet::genesis_config]
+	#[derive(frame_support::DefaultNoBound)]
```

Comment:
why do we need `DefaultNoBound` ?

Reply:
`std` was removed and `GenesisConfig` needed to implement Default and something else required the `DefaultNoBound`

Reply:
Additionally, this was taken from the Substrate repo 1.0.0 updates.